app/controllers/provider_interface/organisations_controller.rb

```diff
@@ -12,8 +12,8 @@ def show
       permissions = scope.where(training_provider: manageable_providers)
         .or(scope.where(ratifying_provider: manageable_providers))
```

Comment:
We're still getting every manageable provider! I think replacing `manageable_providers` with `params[:id]` (or equivalent) ought to fix it

Reply:
What would be better 🤔 
```
permissions ||= ProviderRelationshipPermissions.where(training_provider: params[:id])
                .or(ProviderRelationshipPermissions.where(ratifying_provider: params[:id]))
                .includes(:training_provider, :ratifying_provider)

@training_permissions = permissions.where(training_provider_id: params[:id])
@ratifying_permissions = permissions.where(ratifying_provider_id: params[:id])
```
or 
```
@training_permissions = ProviderRelationshipPermissions.where(training_provider_id:                  
       params[:id]).includes(:training_provider)
@ratifying_permissions = ProviderRelationshipPermissions.where(ratifying_provider_id:
       params[:id]).includes(:ratifying_provider)
```

Reply:
I reckon the second one is a bit more readable? As long as the deduplication still works